frame/system/src/weights.rs

```diff
@@ -58,89 +58,99 @@ pub trait WeightInfo {
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: crate::Config> WeightInfo for SubstrateWeight<T> {
 	/// The range of component `b` is `[0, 3932160]`.
-	fn remark(_b: u32, ) -> Weight {
-		Weight::from_ref_time(1_000_000 as u64)
+	fn remark(b: u32, ) -> Weight {
+		Weight::from_ref_time(3_840_000 as u64)
+			// Standard Error: 0
+			.saturating_add(Weight::from_ref_time(362 as u64).saturating_mul(b as u64))
 	}
 	/// The range of component `b` is `[0, 3932160]`.
 	fn remark_with_event(b: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
+		Weight::from_ref_time(14_232_000 as u64)
 			// Standard Error: 0
-			.saturating_add(Weight::from_ref_time(1_000 as u64).saturating_mul(b as u64))
+			.saturating_add(Weight::from_ref_time(1_383 as u64).saturating_mul(b as u64))
 	}
 	// Storage: System Digest (r:1 w:1)
 	// Storage: unknown [0x3a686561707061676573] (r:0 w:1)
 	fn set_heap_pages() -> Weight {
-		Weight::from_ref_time(5_367_000 as u64)
+		Weight::from_ref_time(9_071_000 as u64)
 			.saturating_add(T::DbWeight::get().reads(1 as u64))
 			.saturating_add(T::DbWeight::get().writes(2 as u64))
 	}
 	// Storage: Skipped Metadata (r:0 w:0)
 	/// The range of component `i` is `[1, 1000]`.
 	fn set_storage(i: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 1_000
-			.saturating_add(Weight::from_ref_time(603_000 as u64).saturating_mul(i as u64))
+		Weight::from_ref_time(4_486_000 as u64)
+			// Standard Error: 781
+			.saturating_add(Weight::from_ref_time(609_439 as u64).saturating_mul(i as u64))
+			.saturating_add(T::DbWeight::get().writes(1 as u64))
 			.saturating_add(T::DbWeight::get().writes((1 as u64).saturating_mul(i as u64)))
 	}
 	// Storage: Skipped Metadata (r:0 w:0)
 	/// The range of component `i` is `[1, 1000]`.
 	fn kill_storage(i: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 1_000
-			.saturating_add(Weight::from_ref_time(513_000 as u64).saturating_mul(i as u64))
+		Weight::from_ref_time(4_663_000 as u64)
+			// Standard Error: 914
+			.saturating_add(Weight::from_ref_time(532_359 as u64).saturating_mul(i as u64))
+			.saturating_add(T::DbWeight::get().writes(1 as u64))
```

Comment:
where is this extra write coming from?

Reply:
I think it is the same issue as mentioned https://github.com/paritytech/substrate/pull/12325/files#r977021289

Basically, we do not run the benchmarks when `i = 0`, and thus the "minimum" is 1, but actually it should be base = 0

Reply:
I'll check why this happens.

Reply:
Okay, yep. I can confirm the cause's the same.